CHANGELOG.md

```diff
@@ -6,10 +6,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 
+- fluentd logs are now sent through the collector instead of being sent directly to the backend (#109)
+- Logs are sent through the OpenTelemetry Agent on the local node by default. `otelAgent.enabled` value must be set to `true` when using logs (#127)
 - `otelAgent.ports` and `otelCollector.ports` are selectively enabled depending on what telemetry types are enabled with `metricsEnabled`, `tracesEnabled`, and `logsEnabled`
 
 ### Removed
 
+- Removed `ingestHost`, `ingestPort`, `ingestProtocol`, use `ingestUrl` instead (#123)
+- Removed `logsBackend`, configure `splunk_hec` exporter directly (#123)
+- Removed `splunk.com/index` annotation for logs (#123)
+- Removed `fluentd.config.indexFields` as all fields sent are indexed (#123)
+- Removed `fluentforward` receiver from gateway (#127)
```

Comment:
Wonder if this is desired overall since there may be other users than our td-agent?  Maybe keep for now with a deprecation warning?

Reply:
Seems unlikely. Would rather not encourage its use. I've run into several issues with the receiver. They can always add to their own config.